tests/Backend/Snowflake/CloneIntoWorkspaceTest.php

```diff
@@ -313,7 +313,14 @@ public function testTableAlreadyExistsAndOverwrite()
             self::IMPORT_FILE_PATH
         );
 
-        $bucket = $client2->createBucket('shared-'. uniqid(), 'in');
+        try {
+            $this->_client->dropBucket('out.c-linked-bucket');
```

Comment:
každé musí být ve vlastní try/catch IMHO, jinak pokud z nějakého důvodu failne unlinknutí, tak neproběhne smazání a failne ti založení. A mimochodem máme na to \Keboola\Test\StorageApiTestCase::dropBucketIfExists

Reply:
Jo jasne ja som vedel ze tam nieco take mame :D